style/ember/sample.js

```diff
@@ -16,3 +16,13 @@ test("checks the box", function() {
     ok(checkBox.prop("checked"), "box is checked");
   });
 });
+
+import Ember from 'ember';
+import DS from 'ember-data';
+
+const { computed } = Ember;
+const { attr, hasMany, belongsTo } = DS;
+
+export default DS.Model.extend({
+  ...
```

Comment:
Should we do this for constructors as well?

For example, importing `Ember.Object` would replace JavaScript's `Object`. There are potentially other places that they would collide, but I haven't thought of any.